frontend/app/views/spree/shared/_filters.html.erb

```diff
@@ -11,17 +11,14 @@
           <% labels.each do |nm,val| %>
             <% label = "#{filter[:name]}_#{nm}".gsub(/\s+/,'_') %>
             <li class="nowrap">
-              <input type="checkbox"
-                     id="<%= label %>"
-                     name="search[<%= filter[:scope].to_s %>][]"
-                     value="<%= val %>"
-                     <%= params[:search] && params[:search][filter[:scope]] && params[:search][filter[:scope]].include?(val.to_s) ? "checked" : "" %> />
+            <%= check_box_tag "search[#{filter[:scope].to_s}][]", val,
```

Comment:
This line does not seem to be indented properly. It is inside the `<li>` element so it should have 2 more spaces, right?